frappe/core/doctype/data_import/exporter.py

```diff
@@ -144,6 +143,8 @@ def add_data_row(self, doctype, parentfield, doc, rows, row_idx):
 				if df.fieldtype == "Duration":
 					value = format_duration(flt(value), df.hide_days)
 
+				if df.fieldtype == "Text Editor":
+					value = frappe.utils.strip_html(value)
```

Comment:
Did you test it? If you strip HTML there won't be anything left.

Reply:
Yes I tested it, the text inside the "Text Editor" field still shows in the CSV file.
Added the screenshot in the description

Reply:
Export a text with list in it :smile:

Reply:
html2text is maybe better for export than this (it preserves some structure by converting to closest possible markdown). I sitll don't like either.

Reply:
For exporter it should be fine ig. Data importer's exporter needs to re-importable.

Reply:
html2text preserves the structure. tried and worked.